modules/sandboxed-containers-deleting-catalog-subscription.adoc

```diff
@@ -0,0 +1,21 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-catalog-subscription_{context}"]
+= Deleting the {sandboxed-containers-first} catalog subscription
+
+You can delete the {sandboxed-containers-first} catalog subscription and remove namespace access to the {sandboxed-containers-operator}.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
```

Comment:
`Branch Build` is shown instead of version.

Reply:
Branch Build will display 4.8 when docs are published.